python/py-tensorflow_macos/Portfile

```diff
@@ -107,8 +107,46 @@ if {${name} ne ${subport}} {
         set docdir ${prefix}/share/doc/${subport}
         xinstall -d ${destroot}${docdir}
         xinstall -m 0644 -W ${github_project_path} LICENSE.txt README.md \
-            ${destroot}${docdir}        
+            ${destroot}${docdir}
     }
 
     livecheck.type  none
 }
+
+# This port py-tensorflow_macos (Apple version) is superceded by port
+# py-tensorflow-macos (Tensorflow version).
+# Please replace with this Portfile code when that version
+# becomes more stable and performant.
+# See: https://developer.apple.com/forums/thread/687654
+
+# Obsolete port GitHub apple/tensorflow_macos
+# PortGroup           obsolete 1.0
+#
+# name                py-tensorflow_macos
+# version             0.1
+#
+# platforms           darwin
+# categories-append   lang
+# license             restrictive/distributable
+# maintainers         nomaintainer
+#
+# description         Mac-optimized TensorFlow and TensorFlow Addons
+#
+# long_description    Hardware-accelerated TensorFlow and TensorFlow \
```

Comment:
several of these fields should be removed as well - see the instructions [here](https://trac.macports.org/wiki/PortfileRecipes#replaced-by).

Reply:
Thanks done.